setup.py

```diff
@@ -14,6 +14,7 @@
     author="Patrick Fuller",
     author_email="pat@numat-tech.com",
     packages=['sartorius'],
+    package_data={'sartorius': ['py.typed']},
```

Comment:
I suggest increasing the version so changes are picked up.

Reply:
Yeah, I was going to bump for a PyPI release, but can do it now.